electron/main.js

```diff
@@ -91,6 +91,7 @@ app.on('ready', async () => {
         delete details.requestHeaders[header];
       }
     });
+    delete details.requestHeaders.Origin;
```

Comment:
We can just move it before `onBeforeSendHeaders`, so it doesn't affect the custom `Origin` by developer.

Reply:
Alright. I will check it a little bit later